src/icons/fileIcons.ts

```diff
@@ -1473,5 +1473,18 @@ export const fileIcons: FileIcons = {
       fileNames: ['jsconfig.json'],
       fileExtensions: ['jsconfig.json'],
     },
+    {
+      name: 'ember',
+      fileNames: [
+          '.ember-cli',
+          '.ember-cli.js',
+          'ember-cli-builds.js',
+          'environment.js',
```

Comment:
These fileNames are too generic as they should be shown with the ember icon. It means that there might be lots of projects which are having a `environment.js` file but they are not related to ember.

I'd recomment to remove `environment.js`, `deploy.js`, `fastboot.js` and `targets.js` as they are too generic.

Reply:
Okay. Shame that I can't match by the `config` parent folder.. I'll remove them and make do!